hazelcast/src/main/java/com/hazelcast/client/impl/ClientEngineImpl.java

```diff
@@ -493,27 +507,20 @@ private void callDisconnectionOperation(ClientEndpointImpl endpoint) {
                 // "a disconnected client is reconnected back to same node"
                 return;
             }
-            ClientDisconnectionOperation op = createClientDisconnectionOperation(clientUuid, memberUuid);
-            operationService.run(op);
+            int partitionId = nodeEngine.getPartitionService().getPartitionId(clientUuid);
```

Comment:
what's this `partitionId` for?

Reply:
This should have been deleted. It seems I forget.